cmd/release.go

```diff
@@ -68,6 +70,13 @@ func (c *ReleaseCommand) Init(args []string) error {
 
 // Run implements Command.Run.
 func (c *ReleaseCommand) Run(ctx *cmd.Context) error {
+	if !c.IKWID {
+		fmt.Fprintf(ctx.Stderr, `release-plan is disabled.
+Please contact cmars in #juju on freenode or email juju@lists.ubuntu.com to request someone to release a plan for you.
```

Comment:
Please ask in #juju-partners on freenode or email juju@lists.ubuntu.com ...

Reply:
I'd display this error if the API call failed due to insufficient permissions. -1 on the weird flag.